CMakeLists.txt

```diff
@@ -96,6 +105,8 @@ set(GZ_TOOLS_VER 1)
 # Find Tinyxml2
 gz_find_package(TINYXML2 REQUIRED PRIVATE PRETTY tinyxml2)
 
+gz_find_package(DL REQUIRED PRETTY tinyxml2)
```

Comment:
I'm guessing this is coming from the relocatable installation PR? Also, I don't think the name should be `tinyxml2`.

Reply:
Correct on both counts.  I copied in some of the relocatable bits here, because I was using them to find the descriptor and message installation paths.